File: rotatearray/ro.py
# Starter Code
class Solution:
    def rotate_array(self, arr, n,k):
        k_step_rotation=[0]
        #Write your code here without removing the exising code
	    #the variable 'arr' contains array of integers of size n and integer k.
	    #modified the array 'k_step_rotation' return the output of the program.
        return k_step_rotation


if __name__ == '__main__':
    n = int(input())
    str = input()
    arr = [int(int(i)) for i in str.strip('[]').split(',')]
    x = Solution()
    k= int(input())
    ans=x.rotate_array(arr,n,k)
    for i in range(0, len(ans)):
        print(ans[i], end=" ")

File: rotatearray/ro.java

	
	public static void main(String[] args)
    {
			Scanner sc=new Scanner(System.in);
            int n = sc.nextInt(); 
			int []arr= Arrays.stream(sc.next().split("[,\\[\\]]")).filter(w-> !w.equals("")).mapToInt(Integer::parseInt).toArray();
		    Solution g=new Solution();
            int k = sc.nextInt();
            int []ans=g.rotate_array(arr,n,k);  
            for(int i=0;i<n;i++)
            System.out.print(ans[i] + " ");  
          
	}
}
class Solution
{
	public static int[] rotate_array(int arr[],int n,int k)
        {
            int[] k_step_rotation = new int[n];
			//Write your code here without removing the exising code
	        //the variable 'arr' contains array of integers of size n and integer k.
	        //modified the array 'k_step_rotation' return the output of the program.
			return k_step_rotation;
        }
}